mslice/widgets/workspacemanager/workspacemanager.py

```diff
@@ -56,6 +58,9 @@ def current_list(self):
     def change_tab(self, tab):
         self.tabWidget.setCurrentIndex(tab)
 
+    def highlight_tab(self, tab):
+        self.tabWidget.setTabText(tab, self.tabWidget.tabText(tab) + '*')
```

Comment:
Could we add a check here to see if the tab text already has an `*` in it. At the moment, if you create multiple cuts, it keeps adding a star each time. I think just one star is enough to show that a workspace has been added to that tab.